nautobot/dcim/forms.py

```diff
@@ -5157,3 +5158,111 @@ class Meta:
             "weight",
             "tags",
         )
+
+
+#
+# Virtual Device Context
+#
+
+
+class VirtualDeviceContextForm(NautobotModelForm):
+    device = DynamicModelChoiceField(
+        queryset=Device.objects.all(),
+    )
+    tenant = DynamicModelChoiceField(
+        queryset=Tenant.objects.all(),
+        required=False,
+    )
+    interfaces = DynamicModelMultipleChoiceField(
+        queryset=Interface.objects.all(), required=False, query_params={"device": "$device"}
```

Comment:
This is an invalid python code, as `device` is a class attribute, and the only way to reference it would be `VirtualDeviceContextForm.device` which is also invalid in this context.

i.e
```python
queryset=Interface.objects.filter(device= VirtualDeviceContextForm.device), required=False, query_params={"device": "$device"}
```